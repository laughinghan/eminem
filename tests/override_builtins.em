let .p println
let .fix 1
let .println 2
p.this.should.all.print.correctly.on.many.lines
