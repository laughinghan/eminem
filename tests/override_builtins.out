this
should
all
print
correctly
on
many
lines
0
<null>
